StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+API.swift

```diff
@@ -632,7 +637,9 @@ extension PaymentSheet {
                 params.paymentMethodOptions = paymentMethodoptions
                 paymentMethodType = paymentMethodParams.type
             }
-
+            if let shouldSetAsDefaultPM = shouldSetAsDefaultPM {
+                params.setAsDefaultPM = NSNumber(value: shouldSetAsDefaultPM)
+            }
```

Comment:
Can shorten this to `params.setAsDefaultPM = NSNumber(value: shouldSetAsDefaultPM ?? false)`, same with in the other spot we do this.